albumentations/core/composition.py

```diff
@@ -116,6 +117,26 @@ def set_deterministic(self, flag, save_key="replay"):
         for t in self.transforms:
             t.set_deterministic(flag, save_key)
 
+    def set_mask_interpolation(self, mask_interpolation):
+        if mask_interpolation not in {
+            cv2.INTER_NEAREST,
+            cv2.INTER_NEAREST_EXACT,
+            cv2.INTER_LINEAR,
+            cv2.INTER_LINEAR_EXACT,
+            cv2.INTER_AREA,
+            cv2.INTER_CUBIC,
+            cv2.INTER_LANCZOS4,
+            cv2.INTER_MAX,
+        }:
+            raise ValueError(
+                f"Value {mask_interpolation} is not supported. "
+                f"Choose one of the following methods: cv2.INTER_NEAREST, cv2.INTER_NEAREST_EXACT, cv2.INTER_LINEAR, "
+                f"cv2.INTER_LINEAR_EXACT, cv2.INTER_AREA, cv2.INTER_CUBIC, cv2.INTER_LANCZOS4, cv2.INTER_MAX"
+            )
```

Comment:
Maybe better to create an Enum as class field for this flag?
With Enum error message would be much more clearly.